.github/workflows/publish.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [macos-latest, ubuntu-latest, windows-latest]
+        platform: [macos-latest, ubuntu-latest, windows-2019, windows-2022]
```

Comment:
Are you sure this will build 2 distinctly names `.msi` installers and not two with the same name that will race and only one will effectively be attached to the release?

Reply:
Did this discussion resolve?
I don't have any further comments, will approve after this one :)

Reply:
I spoke with the tauri team about this, and it looks like the action does not support naming the assets directly, so we will not be able to build for two windows builds right now due to this. 

The reason i added the two windows installs is due to the fact GitHub is updating `windows-latest` to windows-2022 here soon. Due to this windows-latest will not build for windows 10 here in the near future.

I am going to revert it to only build windows-latest, but we may want to figure out a solution sooner rather than later.

Reply:
I remember we had issues with Windows Server 2019 builds on Windows 11, so ultimately we want to have multiple builds regardless. Can you create an issue upstream and add a TODO to our action pointing to it so we can follow updates there?

Reply:
Yes, i will get an issue created upstream & add the todo to the workflow.